src/Native/Runtime/unix/UnixContext.cpp

```diff
@@ -467,27 +483,29 @@ void UnwindCursorToRegDisplay(unw_cursor_t *cursor, unw_context_t *unwContext, R
     MCREG_R1(nativeContext->uc_mcontext) = arg1Reg;
 
 #elif defined(_ARM64_)
+
 #define ASSIGN_CONTROL_REGS  \
     ASSIGN_REG(Pc, IP)
-    // ASSIGN_REG(Sp, SP)    \
-    // ASSIGN_REG(Fp, FP)    \
-    // ASSIGN_REG(Lr, LR)    \
+//    ASSIGN_REG(Sp, SP)    \
```

Comment:
This will need to get uncommented to make ARM64 actually work. Does this fail to compile when it is uncommented?

Reply:
I wasn't unsure if I was "allowed" to add PC to libunwind like it is on the latest revision of the libunwind official repository so I decided to comment that section and proceed with the next compilation error.

I have now added it and I have also uncommented that part of the code.

Reply:
> libunwind like it is on the latest revision of the libunwind official repository 

I do not see PC in the latest version of libunwind repository: https://github.com/llvm-mirror/libunwind/blob/master/include/libunwind.h#L524 . Where is it?

Reply:
My mistake, I was looking at the original libunwind's continuation, not LLVM's version: https://github.com/libunwind/libunwind/blob/master/include/libunwind-aarch64.h

I do believe it's index 32 in any case.

Reply:
These constants are typically not used as unchecked indices. I doubt that adding the constant without anything else is going to work.

Reply:
I've looked further into this and it looks like indeed that the PC constant is not required when using LLVM's libunwind. I'll get rid of it.